.htaccess

```diff
@@ -54,10 +56,11 @@
 	RewriteRule ^(.*)/howto/pagination$ /$1/developer_guides/templates/how_tos/pagination [R=301,L]
 	RewriteRule ^(.*)/howto/simple-contact-form$ /$1/developer_guides/forms/how_tos/simple_contact_form [R=301,L]
 	
-	RewriteRule ^([a-z]{2})/installation/?$ /$1/getting_started/installation [R=301,L]
+	RewriteRule ^(.*)/installation/?$ /$1/getting_started/installation [R=301,L]
```

Comment:
This one was already in .htaccess. I only changed the [a-z]{2} to (.*) for consistency :)